core/math/projection.cpp

```diff
@@ -808,6 +884,10 @@ bool Projection::is_orthogonal() const {
 	return columns[3][3] == 1.0;
 }
 
+bool Projection::is_reversed_z() const {
+	return columns[2][2] > 0;
```

Comment:
The situation of far < near is not considered

Reply:
I don't think so either, I think we protect that on `Camera3D` anyway but if not, it's an acceptable assumption.